HHS/Head-Start-TTADP
# [TTAHUB-2167] Goal source with new TR

## Description of change

This ticket has two tasks:

1. Change goal source 'Training event follow-up' to 'Training event'.
2. Prevent changing the goal source on a goal that has been created via a Training Report.

## How to test

1. Make sure the goals sources are updated.
2. Make sure once a goal is created via 'tr' we cannot change the source to anything but 'Training event'.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2167


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
